_posts/2020-02-27-consumption-happiness.md

```diff
@@ -0,0 +1,73 @@
+
+There is a lot of fun to be had here! I found how to minimize my environmental impact while also saving time and cash.
+How you may ask? Are you going to try to sell me something? No, this isn't that kind of blog.
+
+The conveniences providing my life with very minor benefits can easily be removed without noticing.
+Before I get into the nuts and bolts of this *need what you are getting into the bolts* let me
+first start off by saying that I don't sweat the small stuff.
+
+I prefer to see what works and then spread the word on major (simple - another word, like achievable) ways to reduce my carbon footprint.  What follows are a few of my major findings
+
+## The list of Things to Work on
+
+Here are the things to think about and ways to think about reducing these area:
+
+  1. **Consumption**.
+
+      a. Food:
+        - If you think of meat as precious yet limited flavoring, plants become a major source of protein, vitamins, and minerals. Cows and other animals are inefficient to grow for the same caloric and nutritional density. Also, cows fart... a lot.
+
+        - Refrigeration takes lots of energy but dried, pickled and sugared fruit and vegetables are all ways to have plant based food that you can eat any time and without needing to use energy to keep it edible.
+
+      b. **Digital over physical**.
+
+        - Books use paper, you get the idea. Anything you can digitize you should. Lugging more stuff requires more resources
+        - This leaves space for the few things that you physically cannot do without. That book signed by the author you waited in line for-- keep it.  But that book whose text you love... the words will stay with you without the paper.
+
+      c. Buy second hand, or better yet, find free off of websites like freecycle.org (after pandemics yes yes)
+
+      d. When you buy it new, buy quality items, what ever the price, that last a long time. [Buy it for life or long periods](https://www.reddit.com/r/BuyItForLife/) of time.
+
+  2. **Transportation** How are you getting places?
+
+      a. Try bikes, carpooling, walking, public transport.
+      b. Having a car is expensive, so if you buy one, choose a fuel efficient used car. Electric is overrated but the cause is nice.
+
+  3. **Energy usage**. How are you getting power for your home, business, etc.? You could chase out electric demons quite easily by:
+
+      a. Measuring power consumption and reducing it further
+
+      b. Open up windows, lighting areas with skylights, use natural light more often
+
+      c. Further reading is [chasing electric demons](https://www.mrmoneymustache.com/2015/03/25/cut-your-power-bill/)
+
+      d. Wear warm clothes during the winter and have a passive cooling system within the house. If you heat or cool the individual, not the house, you reduce your footprint since the energy usage goes way down.
+
+  4. **Choose high quality leisure activities** that are environmentally friendly.
```

Comment:
I would develop this point more -- there is opportunity to go over the happiness portion of the article here.